src/Libraries/Nop.Data/Migrations/UpgradeTo460/Domain/ProductAttributeCombination450.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Nop.Core;
+using Nop.Core.Domain.Catalog;
+
+namespace Nop.Data.Migrations.UpgradeTo460.Domain
+{
+    public class ProductAttributeCombination450 : ProductAttributeCombination
```

Comment:
I don't like this idea, it's too complicated, it's better to leave this field in the original class and add a description in a comment that the field is no longer used, or move this field to a partial class and still add a comment that the field is left for backwards compatibility during migration